src/components/Home.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+//import './App.css';
+//import './index.css';
+
+function Home() {
+  return (
+      <div className='Homepage overflow-hidden justify-center'>
+        
+        <center>
+          <div className='bg-blue w-100 h-[690px] grid-cols-1 z-10 -mt-72  items-center border-b-[22px] border-orange overflow-x-hidden'>

```

Comment:
I'm not sure that your z-index settings would do anything here. In general you rarely would use z-index settings if you are laying out your page correctly with flex.

Reply:
Found one more, removed in 4ca8d036e38d516042eb0467f47cf823df96723f